src/components/vote/DelegatesListItem.tsx

```diff
@@ -50,9 +52,15 @@ export const DelegatesListItem = ({
             })}
         >
             <td className='p-4'>
-                <span className='font-medium dark:text-white'>
-                    {trimAddress(delegate.username() || delegate.address(), 'long')}
-                </span>
+                {isLongAddress ? (
+                    <Tooltip content={username}>
+                        <span className='font-medium dark:text-white'>
+                            {trimAddress(username, 'long', 'end')}
+                        </span>
```

Comment:
Since we are using a regular truncate, wouldn't it be better to use CSS to truncate the text? One advantage would be that it would be dynamic. It's a bit tricky to achieve in a td cell, but it can be done.

You could use useIsTruncated to optionally add the tooltip.

I'm not marking it as a request for changes, but rather to discuss.